docs/en/TOC.md

```diff
@@ -18,6 +18,7 @@
     - [Cache Co-locality](samples/data_co_locality.md)
   + Operation
     - [Data Preloading](samples/data_warmup.md)
+    - [Scaling Dataset Manualy](samples/dataset_scaling.md)
```

Comment:
How about `Cache runtime manual scaling` ?

Reply:
I think you are right. Actually we are scaling the AlluxioRuntime in this document, and it leads to scaling the related Dataset. Do I need to change the document's title (both en & zh) and the title in zh TOC?

Reply:
Maybe `manually`? `manual` is an adjective, not an adverb.